textdb/textdb-dataflow/src/main/java/edu/uci/ics/textdb/dataflow/queryrewriter/QueryRewriter.java

```diff
@@ -0,0 +1,68 @@
+package edu.uci.ics.textdb.dataflow.queryrewriter;
+
+import edu.uci.ics.textdb.api.common.*;
+import edu.uci.ics.textdb.api.dataflow.IOperator;
+import edu.uci.ics.textdb.common.field.DataTuple;
+import edu.uci.ics.textdb.common.field.StringListField;
+
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Created by kishorenarendran on 25/04/16.
+ */
+public class QueryRewriter implements IOperator{
```

Comment:
I am unsure of what else, it could be called, after a discussion with @chenlica, we had settled on this. What do you think could be a better alternative?